src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -89,8 +89,8 @@ public static ImagePixelExtractingEstimator ExtractPixels(this TransformsCatalog
         /// <seealso cref= "ImageLoadingEstimator" />
         /// </remarks >
         /// <param name="catalog">The transform's catalog.</param>
-        /// <param name="inputColumnName">Name of the input column.</param>
-        /// <param name="outputColumnName">Name of the resulting output column.</param>
+        /// <param name="inputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
+        /// <param name="outputColumnName">Name of column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
```

Comment:
>outputColumnName [](start = 25, length = 16)

you need to switch input and output column. Output column is resulting and input column is name of column to transform.

Reply:
This is how it suppose to be.
```
 /// <param name="outputColumnName"> Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
/// <param name="inputColumnName"> Name of column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
```
And now check what do you have in `param name=`

---
In reply to: [255809779](https://github.com/dotnet/machinelearning/pull/2432#discussion_r255809779) [](ancestors = 255809779)